### DISCUSSION TITLE
Using the stock checker on a website page

### DISCUSSION THREAD

**Discussion by SionHuws99** [jef/streetmerchant#2308] [Help]
Hi Everyone.

Me and a few friends are creating a next gen console stock checker website for college and was wondering if its possible to have the information collected by this source displayed on a web page? This won't be an actual live website and will only be used for educational purposes. If anyone can confirm if this is possible or not and point us in the right direction?

Thanks!

**Comment by 0000day** [comment]
Well, clone the project and do a global search for `logger.info(`, `logger.warn(` and `logger.error(`. Around these lines the informations get set up and logged to the console.

Just rewrite it to send the information where you want it to be. HTTP-Post somewhere?

**Reply by SionHuws99** [reply]
Okay I will look into this! Thank you for your help.
